custom_components/localtuya/light.py

```diff
@@ -36,6 +39,56 @@
 DEFAULT_LOWER_BRIGHTNESS = 29
 DEFAULT_UPPER_BRIGHTNESS = 1000
 
+WHITE_MODE = "white"
```

Comment:
Can we revert the name order here to be consistent with Home Assistant, e.g. `MODE_WHITE` or even `COLOR_MODE_WHITE`?

Reply:
Done.